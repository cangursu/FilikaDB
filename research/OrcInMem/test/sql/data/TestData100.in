0,cln-0,0
1,cln-1,0.1
2,cln-2,0.2
3,cln-3,0.3
4,cln-4,0.4
5,cln-5,0.5
6,cln-6,0.6
7,cln-7,0.7
8,cln-8,0.8
9,cln-9,0.9
10,cln-10,1
11,cln-11,1.1
12,cln-12,1.2
13,cln-13,1.3
14,cln-14,1.4
15,cln-15,1.5
16,cln-16,1.6
17,cln-17,1.7
18,cln-18,1.8
19,cln-19,1.9
20,cln-20,2
21,cln-21,2.1
22,cln-22,2.2
23,cln-23,2.3
24,cln-24,2.4
25,cln-25,2.5
26,cln-26,2.6
27,cln-27,2.7
28,cln-28,2.8
29,cln-29,2.9
30,cln-30,3
31,cln-31,3.1
32,cln-32,3.2
33,cln-33,3.3
34,cln-34,3.4
35,cln-35,3.5
36,cln-36,3.6
37,cln-37,3.7
38,cln-38,3.8
39,cln-39,3.9
40,cln-40,4
41,cln-41,4.1
42,cln-42,4.2
43,cln-43,4.3
44,cln-44,4.4
45,cln-45,4.5
46,cln-46,4.6
47,cln-47,4.7
48,cln-48,4.8
49,cln-49,4.9
50,cln-50,5
51,cln-51,5.1
52,cln-52,5.2
53,cln-53,5.3
54,cln-54,5.4
55,cln-55,5.5
56,cln-56,5.6
57,cln-57,5.7
58,cln-58,5.8
59,cln-59,5.9
60,cln-60,6
61,cln-61,6.1
62,cln-62,6.2
63,cln-63,6.3
64,cln-64,6.4
65,cln-65,6.5
66,cln-66,6.6
67,cln-67,6.7
68,cln-68,6.8
69,cln-69,6.9
70,cln-70,7
71,cln-71,7.1
72,cln-72,7.2
73,cln-73,7.3
74,cln-74,7.4
75,cln-75,7.5
76,cln-76,7.6
77,cln-77,7.7
78,cln-78,7.8
79,cln-79,7.9
80,cln-80,8
81,cln-81,8.1
82,cln-82,8.2
83,cln-83,8.3
84,cln-84,8.4
85,cln-85,8.5
86,cln-86,8.6
87,cln-87,8.7
88,cln-88,8.8
89,cln-89,8.9
90,cln-90,9
91,cln-91,9.1
92,cln-92,9.2
93,cln-93,9.3
94,cln-94,9.4
95,cln-95,9.5
96,cln-96,9.6
97,cln-97,9.7
98,cln-98,9.8
99,cln-99,9.9
