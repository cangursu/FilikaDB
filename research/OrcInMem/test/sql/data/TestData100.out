{"col1": 0, "col2": "cln-0", "col3": 0}
{"col1": 1, "col2": "cln-1", "col3": 0.1}
{"col1": 2, "col2": "cln-2", "col3": 0.2}
{"col1": 3, "col2": "cln-3", "col3": 0.3}
{"col1": 4, "col2": "cln-4", "col3": 0.4}
{"col1": 5, "col2": "cln-5", "col3": 0.5}
{"col1": 6, "col2": "cln-6", "col3": 0.6}
{"col1": 7, "col2": "cln-7", "col3": 0.7}
{"col1": 8, "col2": "cln-8", "col3": 0.8}
{"col1": 9, "col2": "cln-9", "col3": 0.9}
{"col1": 10, "col2": "cln-10", "col3": 1}
{"col1": 11, "col2": "cln-11", "col3": 1.1}
{"col1": 12, "col2": "cln-12", "col3": 1.2}
{"col1": 13, "col2": "cln-13", "col3": 1.3}
{"col1": 14, "col2": "cln-14", "col3": 1.4}
{"col1": 15, "col2": "cln-15", "col3": 1.5}
{"col1": 16, "col2": "cln-16", "col3": 1.6}
{"col1": 17, "col2": "cln-17", "col3": 1.7}
{"col1": 18, "col2": "cln-18", "col3": 1.8}
{"col1": 19, "col2": "cln-19", "col3": 1.9}
{"col1": 20, "col2": "cln-20", "col3": 2}
{"col1": 21, "col2": "cln-21", "col3": 2.1}
{"col1": 22, "col2": "cln-22", "col3": 2.2}
{"col1": 23, "col2": "cln-23", "col3": 2.3}
{"col1": 24, "col2": "cln-24", "col3": 2.4}
{"col1": 25, "col2": "cln-25", "col3": 2.5}
{"col1": 26, "col2": "cln-26", "col3": 2.6}
{"col1": 27, "col2": "cln-27", "col3": 2.7}
{"col1": 28, "col2": "cln-28", "col3": 2.8}
{"col1": 29, "col2": "cln-29", "col3": 2.9}
{"col1": 30, "col2": "cln-30", "col3": 3}
{"col1": 31, "col2": "cln-31", "col3": 3.1}
{"col1": 32, "col2": "cln-32", "col3": 3.2}
{"col1": 33, "col2": "cln-33", "col3": 3.3}
{"col1": 34, "col2": "cln-34", "col3": 3.4}
{"col1": 35, "col2": "cln-35", "col3": 3.5}
{"col1": 36, "col2": "cln-36", "col3": 3.6}
{"col1": 37, "col2": "cln-37", "col3": 3.7}
{"col1": 38, "col2": "cln-38", "col3": 3.8}
{"col1": 39, "col2": "cln-39", "col3": 3.9}
{"col1": 40, "col2": "cln-40", "col3": 4}
{"col1": 41, "col2": "cln-41", "col3": 4.1}
{"col1": 42, "col2": "cln-42", "col3": 4.2}
{"col1": 43, "col2": "cln-43", "col3": 4.3}
{"col1": 44, "col2": "cln-44", "col3": 4.4}
{"col1": 45, "col2": "cln-45", "col3": 4.5}
{"col1": 46, "col2": "cln-46", "col3": 4.6}
{"col1": 47, "col2": "cln-47", "col3": 4.7}
{"col1": 48, "col2": "cln-48", "col3": 4.8}
{"col1": 49, "col2": "cln-49", "col3": 4.9}
{"col1": 50, "col2": "cln-50", "col3": 5}
{"col1": 51, "col2": "cln-51", "col3": 5.1}
{"col1": 52, "col2": "cln-52", "col3": 5.2}
{"col1": 53, "col2": "cln-53", "col3": 5.3}
{"col1": 54, "col2": "cln-54", "col3": 5.4}
{"col1": 55, "col2": "cln-55", "col3": 5.5}
{"col1": 56, "col2": "cln-56", "col3": 5.6}
{"col1": 57, "col2": "cln-57", "col3": 5.7}
{"col1": 58, "col2": "cln-58", "col3": 5.8}
{"col1": 59, "col2": "cln-59", "col3": 5.9}
{"col1": 60, "col2": "cln-60", "col3": 6}
{"col1": 61, "col2": "cln-61", "col3": 6.1}
{"col1": 62, "col2": "cln-62", "col3": 6.2}
{"col1": 63, "col2": "cln-63", "col3": 6.3}
{"col1": 64, "col2": "cln-64", "col3": 6.4}
{"col1": 65, "col2": "cln-65", "col3": 6.5}
{"col1": 66, "col2": "cln-66", "col3": 6.6}
{"col1": 67, "col2": "cln-67", "col3": 6.7}
{"col1": 68, "col2": "cln-68", "col3": 6.8}
{"col1": 69, "col2": "cln-69", "col3": 6.9}
{"col1": 70, "col2": "cln-70", "col3": 7}
{"col1": 71, "col2": "cln-71", "col3": 7.1}
{"col1": 72, "col2": "cln-72", "col3": 7.2}
{"col1": 73, "col2": "cln-73", "col3": 7.3}
{"col1": 74, "col2": "cln-74", "col3": 7.4}
{"col1": 75, "col2": "cln-75", "col3": 7.5}
{"col1": 76, "col2": "cln-76", "col3": 7.6}
{"col1": 77, "col2": "cln-77", "col3": 7.7}
{"col1": 78, "col2": "cln-78", "col3": 7.8}
{"col1": 79, "col2": "cln-79", "col3": 7.9}
{"col1": 80, "col2": "cln-80", "col3": 8}
{"col1": 81, "col2": "cln-81", "col3": 8.1}
{"col1": 82, "col2": "cln-82", "col3": 8.2}
{"col1": 83, "col2": "cln-83", "col3": 8.3}
{"col1": 84, "col2": "cln-84", "col3": 8.4}
{"col1": 85, "col2": "cln-85", "col3": 8.5}
{"col1": 86, "col2": "cln-86", "col3": 8.6}
{"col1": 87, "col2": "cln-87", "col3": 8.7}
{"col1": 88, "col2": "cln-88", "col3": 8.8}
{"col1": 89, "col2": "cln-89", "col3": 8.9}
{"col1": 90, "col2": "cln-90", "col3": 9}
{"col1": 91, "col2": "cln-91", "col3": 9.1}
{"col1": 92, "col2": "cln-92", "col3": 9.2}
{"col1": 93, "col2": "cln-93", "col3": 9.3}
{"col1": 94, "col2": "cln-94", "col3": 9.4}
{"col1": 95, "col2": "cln-95", "col3": 9.5}
{"col1": 96, "col2": "cln-96", "col3": 9.6}
{"col1": 97, "col2": "cln-97", "col3": 9.7}
{"col1": 98, "col2": "cln-98", "col3": 9.8}
{"col1": 99, "col2": "cln-99", "col3": 9.9}